sdks/python/apache_beam/io/gcp/bigquery_file_loads_test.py

```diff
@@ -716,8 +826,10 @@ def test_multiple_destinations_transform(self):
               max_file_size=20,
               max_files_per_bundle=-1))
 
+  @parameterized.expand(
```

Comment:
I was not able to run this with Dataflow runner. I can revert it back if more appropriate.

Reply:
I reverted the test as I still couldn't run it successfully even with local Dataflow runner. I will follow up in a separate PR if needed.